vscode-dotnet-runtime-extension/package.json

```diff
@@ -4,26 +4,70 @@
 		"type": "git",
 		"url": "https://github.com/dotnet/vscode-dotnet-runtime.git"
 	},
+	"bugs": {
+		"url": "https://github.com/dotnet/vscode-dotnet-runtime/issues"
+	},
 	"license": "MIT",
-	"description": "Allows acquisition of the .NET runtime specifically for VSCode extension authors.",
+	"author": "Microsoft Corporation",
+	"displayName": ".NET Install Tool for Extension Authors",
+	"description": "Allows acquisition of the .NET runtime specifically for VS Code extension authors.",
+	"icon": "images/dotnetIcon.png",
+	"version": "1.0.0",
+	"publisher": "ms-dotnettools",
 	"engines": {
 		"vscode": "^1.41.0"
 	},
-	"version": "1.0.0",
-	"publisher": "ms-dotnettools",
+	"categories": [
+		"Other"
+	],
+	"keywords": [
+		".NET",
+		".NET Core",
+		"dotnet",
+		"Extension Authoring"
+	],
 	"activationEvents": [
 		"onCommand:dotnet.acquire",
 		"onCommand:dotnet.uninstallAll",
 		"onCommand:dotnet.showAcquisitionLog",
-		"onCommand:dotnet.ensureDotnetDependencies"
+		"onCommand:dotnet.ensureDotnetDependencies",
+		"onCommand:dotnet.reportIssue"
 	],
-	"main": "./out/extension.js",
+	"main": "./dist/extension.js",
+	"contributes": {
+		"commands": [
+			{
+				"command": "dotnet.reportIssue",
```

Comment:
Do you think we'd end up having a separate "report issue" for the SDK extension? Only asking because it's something we may need to start thinking about if it's the path we take so commands don't collide

Reply:
I imagine we'll have to choose a different command prefix. It's unfortunate that we used `dotnet` and not something more specific to this extension, but changing that now would be breaking.

Reply:
> but changing that now would be breaking.

It'd only be breaking if other extensions invoked it though right? I'd be curious if this was actually invoked by other extensions in the current world. At the very least Google doesn't seem to think any other OSS extension uses it: https://www.google.com/search?q=%22dotnet.reportIssue%22&rlz=1C1CHBF_enUS892US892&oq=%22dotnet.reportIssue%22&aqs=chrome..69i57j33i299.1498j0j7&sourceid=chrome&ie=UTF-8

Reply:
Oh I guess that's true, I don't think any extensions would call it. Although just from pure aesthetics I don't love the idea of command prefixes not being consistent across the extension.